iothub_client/samples/ios/CocoaPods-Samples.md

```diff
@@ -43,7 +43,7 @@ The `azure-iot-sdk-c/iothub_client/samples` directory some Azure IoT samples tha
 are appropriate for working with the iOS sample application:
 
 * **iothub_ll_c2d_sample** (cloud to device) This sample demonstrates sending messages
-from the cloud to the device. Device Explorer can be used to send the messages.
+from the cloud to the device. Azure IoT Explorer can be used to send the messages.
```

Comment:
nit, should this be a link to the github and/or referenced as recommended in the prereqs for the doc?

Reply:
I am just following the instructions we received from the team. I forwarded you the e-mail as FYI,